public/Day 61/src/client/.env.example

```
@@ -0,0 +1,11 @@
+# -------------------------
+# API ENDPOINTS
+# -------------------------
+NEXT_PUBLIC_API_URL=         # Base API URL (used by Next.js, accessible on both client & server)
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)